app/Console/Kernel.php

```diff
@@ -96,6 +97,11 @@ protected function schedule(Schedule $schedule): void
 
     private function scheduleJobsForCollectionsOrGalleries(Schedule $schedule): void
     {
+        $schedule
+            ->command(FetchCollectionActivity::class)
+            ->withoutOverlapping()
+            ->dailyAt('10:00');
```

Comment:
I don't think we need to do this daily for all collections. I'd start with weekly and we should fetch it more often for collections where we notice people more often view them or if they are more popular in general (most owners in db, some other popularity metric, etc) . Out of scope for this PR, so I'd just start with weekly updates for now